src/main/java/com/shipmentEvents/demo/TicTacToe.java

```diff
@@ -0,0 +1,166 @@
+package com.company;
+
+import java.util.Scanner;
+
+
+public class TicTacToe {
+
+    public static void main(String[] args) {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The cyclomatic complexity of this method is 15. By comparison, 99% of the methods in the CodeGuru reference dataset have a lower cyclomatic complexity. This indicates the method has a high number of decisions and it can make the logic difficult to understand and test. We recommend that you simplify this method or break it into multiple methods.